pkg/app/piped/executor/ecs/rollback.go

```diff
@@ -105,7 +105,7 @@ func rollback(ctx context.Context, in *executor.Input, platformProviderName stri
 	// Re-register TaskDef to get TaskDefArn.
 	// Consider using DescribeServices and get services[0].taskSets[0].taskDefinition (taskDefinition of PRIMARY taskSet)
 	// then store it in metadata store and use for rollback instead.
-	td, err := client.RegisterTaskDefinition(ctx, taskDefinition)
+	td, err := client.RegisterTaskDefinition(ctx, taskDefinition, serviceDefinition.Tags)
```

Comment:
I think we only care about the cluster/service/task thus we don't need to tag the taskDefinition, since it isn't a part of the application anyway. wdyt?

Reply:
I can also get TaskDef from Task, and it certainly seems unnecessary...
I'll give it some more thought and fix it!🙇

Reply:
Any besides, we should check whether tags which are added to the service is automatically added to the tasks which are controlled by service or not as well 👀

Reply:
The `--propagateTags` parameter can be used to copy tags from a task definition or service to a task in the service🚀